README.md

```diff
@@ -216,7 +216,7 @@ The default values are shown after each option key.
 	timeout: 0,         // req/res timeout in ms, it resets on redirect. 0 to disable (OS limit applies)
 	compress: true,     // support gzip/deflate content encoding. false to disable
 	size: 0,            // maximum response body size in bytes. 0 to disable
-	agent: null         // http(s).Agent instance, allows custom proxy, certificate etc.
+	agent: null         // http(s).Agent instance, allows custom proxy, certificate, lookup, family etc.
```

Comment:
Maybe only add `lookup`? I’m still not sure if family actually works with agent

Reply:
I just added a test proving that it does indeed work.